odoo/addons/base/rng/common.rng

```diff
@@ -387,4 +387,25 @@
         </rng:element>
     </rng:define>
 
+    <!--
+        NOTE: right now we don't need to support overload, access_rights and modifiable for two the following tags.
```

Comment:
I would keep at least the following mixin on both `control` and `create` , to allow the simple form of inheritance:
            <rng:ref name="overload"/>